src/functions/createProduct/index.ts

```diff
@@ -0,0 +1,13 @@
+import { handlerPath } from '@libs/handlerResolver';
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      http: {
+        method: 'post',
+        path: 'products',
+      }
+    }
+  ]
+}
```

Comment:
Post request returns error
<img width="1033" alt="Screenshot 2021-08-02 at 18 09 57" src="https://user-images.githubusercontent.com/85166028/127883512-08f7d0b0-7b9f-470b-bdb4-5b2b35645817.png">

Reply:
thanks. My mistake. it seems I was sending json string during my tests